app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```
@@ -1,68 +1,30 @@
+  Tab,
+  Tabs,
   Tooltip,
   Typography,
   useTheme,
 } from "@mui/material";
-import get from "lodash/get";
 import React, { useEffect, useMemo, useState } from "react";
-import { useRecoilState, useSetRecoilState } from "recoil";
 import ActionMenu from "./ActionMenu";
+import { COMPARE_KEY_SECONDARY_COLOR } from "./constants";
 import Error from "./Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)